examples/elm-flags/index.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+
+<body>
+    <div id="flags-example"></div>
+    <script src="elm.js"></script>
+    <script>
+      var node = document.querySelector("#flags-example")
+      // the object passed as the second parameter contains the flags passed to our init function
+      Elm.Main.embed(node, {
+          url: window.location.href
```

Comment:
effect is: 
![image](https://cloud.githubusercontent.com/assets/194400/24365827/1ea3e12e-130f-11e7-96fc-76ee8061311f.png)